content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx

```diff
@@ -150,12 +150,12 @@ const prisma = new PrismaClient()
 // use `prisma` in your application to read and write data in your DB
 ```
 
-#### Keeping the query engine binary out of version control by default
+#### Keeping the query engine out of version control by default
 
-Prisma Client is based on a _query engine_ that's running as a sidecar process alongside your application. This query engine _binary_ is downloaded when `prisma generate` is invoked and stored in the `output` path.
+Prisma Client under the hood uses a [_query engine_](../prisma-engines/query-engine) to run queries against the database. This query engine is downloaded when `prisma generate` is invoked and stored in the `output` path together with the generated Client.
 
-By generating Prisma Client into `node_modules`, the query engine is kept out of version control by default (since `node_modules` is typically ignored for version control). If it was not generated into `node_modules`, then you would need to explicitly ignore it, e.g. for Git you'd need to add the `output` path to your `.gitignore` file.
+By generating Prisma Client into `node_modules`, the query engine is kept out of version control by default (since `node_modules` is typically ignored for version control). If it was not generated into `node_modules`, then you would need to explicitly ignore it, e.g. for Git you'd need to add the `output` path to your `.gitignore` file. <!-- TODO rephrase this -->
```

Comment:
```suggestion
By generating Prisma Client into `node_modules`, the query engine is usually kept out of version control by default since `node_modules` is typically ignored for version control.
When using a custom `output` path for the generated Prisma Client, it is advised to exclude it from your version control. For Git, this means adding the `output` path to your `.gitignore` file.
```
I gave this a try